code/game/jobs/job/blackshield.dm

```diff
@@ -224,3 +225,55 @@
 	name = "Blackshield Trooper"
 	icon_state = "player-blue"
 	join_tag = /datum/job/trooper
+
+
+/datum/job/cadet
+	title = "Blackshield Cadet"
+	flag = BSCADET
+	department = DEPARTMENT_BLACKSHIELD
+	department_flag = BLACKSHIELD
+	faction = MAP_FACTION
+	total_positions = 2
+	spawn_positions = 2
+	supervisors = "the Blackshield Commander"
+	difficulty = "Easy."
+	selection_color = "#a7bbc6"
+	wage = WAGE_LABOUR_INTERN
+	health_modifier = 10
+	disallow_species = list(FORM_SOTSYNTH, FORM_AGSYNTH, FORM_NASHEF)
+
+	outfit_type = /decl/hierarchy/outfit/job/blackshield/troop/cadet
+
+	perks = list(PERK_BLACKSHIELD_CONDITIONING, PERK_BOLT_REFLECT, PERK_CHEM_CONTRABAND)
+
+	access = list(
+		access_security, access_eva,
+		access_sec_doors, access_brig, access_maint_tunnels, access_external_airlocks
+	)
+
+	stat_modifiers = list(
+		STAT_ROB = 25,
+		STAT_TGH = 20,
+		STAT_VIG = 25,
+	)
+
+	software_on_spawn = list(/datum/computer_file/program/digitalwarrant,
+							 /datum/computer_file/program/camera_monitor)
+
+	description = "The Trooper forms the base of the Blackshield, the front line against pirates, terrorists, and xenos.<br>\
+	You are learning to become a Trooper and help protect the colony. Employ your talents to bring an end to threats and conflict situations.<br>\
+	Tactics and teamwork are vital, make sure to learn some. You are paid to follow and learn orders, not to question. Remember your focus on external threats - leave otherwise to Marshals.<br>\
```

Comment:
```suggestion
	Tactics and teamwork are vital, make sure to learn some. You are paid to follow and learn orders, not to question. Remember your focus is on external threats - leave otherwise to Marshals.<br>\
```
Grammar